cluster/gce/util.sh

```diff
@@ -545,6 +545,29 @@ function get-template-name-from-version() {
   echo "${NODE_INSTANCE_PREFIX}-template-${1}" | cut -c 1-63 | sed 's/[\.\+]/-/g;s/-*$//g'
 }
 
+# validates the NODE_LOCAL_SSDS_EXT variable 
+function validate-node-local-ssds-ext(){
+  ssdopts=$1
+
+  if [[ -z ${ssdopts[0]} || -z ${ssdopts[1]} || -z ${ssdopts[2]} ]]; then
+	  echo -e "${color_red}Local SSD: NODE_LOCAL_SSDS_EXT is malformed, found ${ssdopts[0]-_},${ssdopts[1]-_},${ssdopts[2]-_} ${color_norm}" >&2
+    exit 2
+  fi
+  if [[ ${ssdopts[1]} != "scsi" && ${ssdopts[1]} != "nvme" ]]; then
+    echo -e "${color_red}Local SSD: Interface must be scsi or nvme, found: ${ssdopts[1]} ${color_norm}" >&2
+    exit 2
+  fi
+  if [[ ${ssdopts[2]} != "fs" && ${ssdopts[2]} != "block" ]]; then
+    echo -e "${color_red}Local SSD: Filesystem type must be fs or block, found: ${ssdopts[2]} ${color_norm}"  >&2
+    exit 2
+  fi
+  local_ssd_count=$((local_ssd_count+ssdopts[0]))
+  if [[ ${local_ssd_count} -gt 8 || ${local_ssd_count} -lt 1 ]]; then
```

Comment:
Does this also include the count from NODE_LOCAL_SSDS?

Reply:
yep